protos/core/Btfs.proto

```diff
@@ -0,0 +1,230 @@
+syntax = "proto3";
+
+
+
+package btfs;
+
+
+
+
+
+// service EscrowService{. //FYI from Cong's review, this part will be removed eventually
+// 	submitContracts()
+// 	fullPay()
+// 	checkContract()
+// 	cancelContract()
+// }
+
+service GuardService{
+	rpc submitFileStoreMeta(FileStoreMeta) returns (Result){}
+	rpc health() returns(HealthStatus){} //used for other nodes check guard health , TBD input argument
+  	rpc sendQuestions(Questions) returns(Result){} //used for renter send the questions to guard
+  	rpc checkFileStoreMeta(CheckFileStoreMetaRequest) returns (FileStoreMeta){} //reserved for other nodes to check contract status
+}
+
+service HostService{
```

Comment:
Remove. This will be on libp2p and thus will be an http api.

Reply:
done the removal